.jxbrowser.userdata/AutofillStates/2020.11.2.164946/AD

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
remove all .jxbrowser folders

Reply:
- added all .jxbrowser folders to gitignore
- removed all .jxbrowser folders from git tracking